Jenkinsfile

```diff
@@ -30,7 +30,6 @@ pipeline {
   }
   parameters {
     string(name: 'MAVEN_CONFIG', defaultValue: '-B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn', description: 'Additional maven options.')
-    booleanParam(name: 'Run_As_Master_Branch', defaultValue: false, description: 'Allow to run any steps on a PR, some steps normally only run on master branch.')
```

Comment:
Seems unrelated to this change.

Reply:
`Master` is in there, but somehow this particular parameter is not used at all in the pipeline, so rather than changing `master` for `main` , I decided to delete the parameter itself

Reply:
I don't mind to revert this change if it helps

Reply:
gotcha. seems fine to remove then.